.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,20 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+  "features": {
+    "ghcr.io/shyim/devcontainers-features/symfony-cli:0": {},
```

Comment:
Can the Symfony organisation host these images, so that it doesn't depend on an external contributor? (no offence to shyim, but this account may be closed, hacked, etc. and people would end up with no image or a counterfeit image).

Reply:
I don't know if Symfony wants to host some stuff. I am open :) 
You should also be pinning to a specific sha if you prefer

Reply:
@javiereguiluz Would it be possible to add a project to the Symfony organisation in order to build some Docker images? Where (what repo) should we request this?

Reply:
For the Symfony CLI, this should probably be discussed on https://github.com/symfony-cli/symfony-cli/issues (and then, the repo needed for that would also live in that organization, not in the Symfony one)

Reply:
Actually it looks like I was confused by the `ghcr.io` name, `ghcr.io/shyim/devcontainers-features/symfony-cli:0` is not a Docker image as I expected but it's a text recipe. But it would be nice to host it on Symfony's side though, I'll open an issue.